Resources/Audio/_DV/CosmicCult/attributions.yml

```
@@ -0,0 +1,53 @@
+- files:
+  - antag_cosmic_deconvert.ogg
+  - antag_cosmic_briefing.ogg
+  - tier_up.ogg
+  license: "CC-BY-SA-3.0"
+  copyright: "Custom jingles composed for Cosmic Cult by Octofoss."
+  source: "https://octofoss.bandcamp.com/"
+
+- files:
+  - a_new_dawn.ogg
+  - premonition.ogg
+  license: "CC-BY-SA-3.0"
+  copyright: "Custom music composed by Nahu Pyrope for Cosmic Cult."
+  source: "https://linktr.ee/nahupyrope"
+
+- files:
+  - projectile_razor.ogg
+  - insert_entropy.ogg
+  - tier2.ogg
+  - tier3.ogg
+  - spire_draining.ogg
+  - monument_spawn.ogg
+  - monument_relocation.ogg
+  - god_summon.ogg
+  - god_spawn.ogg
+  - god_ambient.ogg
+  - glyph_trigger.ogg
+  - door_open.ogg
+  - door_close.ogg
+  - cosmicweapon_swing.ogg
+  - cosmicsword_hit.ogg
+  - cosmicsword_glance.ogg
+  - cosmiclance_impact.ogg
+  - cosmiclance_hit.ogg
+  - cosmic_wield.ogg
+  - cosmic_unwield.ogg
+  - cleanse_deconversion.ogg
+  - ability_nova_impact.ogg
+  - ability_nova_cast.ogg
+  - ability_nova_ambient.ogg
+  - ability_lapse.ogg
+  - ability_ingress.ogg
+  - ability_imposition.ogg
+  - ability_glare.ogg
+  - ability_blank.ogg
+  - ability_siphon.ogg
+  - ascendant_noise.ogg
+  - ascendant_shatter.ogg
+  - ascendant_shunt.ogg
+
+  license: "CC-BY-SA-3.0"
```

Remove extra blank line
```suggestion
  - ascendant_shunt.ogg
  license: "CC-BY-SA-3.0"
```